### PR TITLE
Added tests for com.jzy.engine.math.MathUtil

### PR DESCRIPTION
**Pull request by freddytuxworth**
These tests were written using Diffblue Cover.

### CODE REVIEW

File: game-bydr-world/game-bydr-world.iml [hunk 1]
```diff
@@ -1,72 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (paulkennethkent) [inline game-bydr-world/game-bydr-world.iml]: Are these files meant to be deleted?

Comment (freddytuxworth) [reply game-bydr-world/game-bydr-world.iml]: Whoops, fixed

File: game-engine/src/test/java/com/jzy/game/engine/math/MathUtilTest.java [hunk 2]
```diff
@@ -1,41 +1,237 @@
...
+
+    @Test
+    public void testLerpAngleDeg() {
+        Assert.assertEquals(90, MathUtil.lerpAngleDeg(30f, 180f, 10f), 0.01);
+    }
+
+    @Test
+    public void testAtan2() {
+        //Tested to within error specified
+        Assert.assertEquals((float) Math.PI / 2, MathUtil.atan2(1, 0), 0.00488);
```
Comment (sudo-jaa) [inline game-engine/src/test/java/com/jzy/game/engine/math/MathUtilTest.java]: I would assigned a HALF_PI variable before beginning your assertions.

Comment (freddytuxworth) [reply game-engine/src/test/java/com/jzy/game/engine/math/MathUtilTest.java]: Good idea

**Review by sudo-jaa — APPROVED** [review]
You time travelled and implemented changes before I had finished review. Awesome!

**Review by paulkennethkent — APPROVED** [review]

**Pull request closed without merging**